tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/test_settings_change_language.py

```diff
@@ -10,17 +10,20 @@
 
 class TestChangeLanguage(GaiaTestCase):
 
+    def setUp(self):
+        GaiaTestCase.setUp(self)
+        self.data_layer.set_setting('devtools.pseudolocalization.enabled', True)
+
     def test_change_language_settings(self):
         lang_name = self.marionette.execute_script("""
-            var  = window.wrappedJSObject.navigator.mozL10n.;
-            return ['fr-x-psaccent'].translate('Packaged Accented');
+            var qps = window.wrappedJSObject.navigator.mozL10n.qps;
```

Comment:
Locale codes starting with "q" can be used to describe custom languages. "ps" is short for "pseudolocalization". For example, Microsoft uses "qps-ploc" as a code for one of their pseudolocalizations.

We're moving away from this scheme in favor of "fr-x-psaccent" and "ar-x-psbidi", where "-x-" is also used as a means of introducing a custom extension.  The reason for this move is that `Intl` and other language negotiation mechanisms give up completely on "qps-ploc";  however, with "ar-x-psbidi", they usually understand that we want something in Arabic and for example can properly format date and time.